content/projects/gsoc/students.adoc

```diff
@@ -12,9 +12,17 @@ See link:/projects/gsoc[the main GSoC project] page for other information and li
 
 == Student Application process
 
+=== Eligibility steps
+
+You must verify that you are eligible to participate in the program.
+We cannot make any exception. Please:
+
+. Make sure you are link:https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation[eligible to participate in the program]
+. If you are an accepted students in the United States of America on an F1 visa, you must obtain an link:https://developers.google.com/open-source/gsoc/faq#i_am_an_accepted_student_in_the_united_states_on_an_f1_visa_how_do_i_get_authorization_to_participate[authorization before you can participate].
+
 === Application steps
 
-. Check out the link:/projects/gsoc/gsoc2018-project-ideas[GSoC 2018 Project ideas]
+. Check out the link:/projects/gsoc/2019/project-ideas[GSoC 2019 Project ideas]
 . Select an interesting project idea or draft your own proposal.
 . If you are not familiar with Jenkins, read the introductory info on the website and try it out with one of your previous projects.
 . Join the _jenkinsci-dev_ and _jenkinsci-gsoc-all-public_ mailing lists, add GSoC office hours to your calendar
```

Comment:
@martinda I suggest you also add:
> Join https://gitter.im/jenkinsci/gsoc-sig

Reply:
I added it to the community bonding period, after talking to mentors.